src/components/radioButton/demoMultiColumn/script.js

```diff
@@ -24,8 +24,8 @@ angular
       'lastName': 'Castel',
       'title': "Security"
     }];
-    self.selectedIndex = 2;
+    self.selectedId = 2;
     self.selectedUser = function() {
-      return self.contacts[self.selectedIndex].lastName;
+      return $filter('filter')(self.contacts, { id: self.selectedId })[0].lastName;
```

Comment:
Nice, looks good (instead of using `Closes` - should be `Fixes` - but that's no problem). I'll mark this to be ready for merge.